### PR TITLE
lmb-946: generate mandatarissen in mandataris service instead of the frontend

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

We moved the logic of generating mandatarissen for the IV to this service. We pass on the start, end ,mandaat, rangorde on to the call and we add the default statussen in the query itself.

## How to test

Run this branch with the frontend and look at the created data if its correct.

## Links to other PR's

- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/413

### CODE REVIEW

File: util/rangorde.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+function getRangordeNumberMapping() {
+  const numberWithString = {};
+  Object.keys(rangordeStringMapping).forEach((key) => {
+    numberWithString[rangordeStringMapping[key]] = key;
+  });
+
+  return numberWithString;
+}
+
+const rangordeStringMapping = {
```
Comment (JonasVanHoof) [inline util/rangorde.ts:16]: Copied this over from the frontend...

File: data-access/mandataris.ts [hunk 2]
```diff
@@ -833,3 +839,65 @@ async function getMandatarisFracties(
...
+        sparqlEscapeUri(item.uri),
+        sparqlEscapeString(item.id),
+        sparqlEscapeString(item.rangorde),
+      ];
+
+      return `( ${values.join(' ')} )`;
+    })
+    .join('\n');
+
+  const escapedCommon = {
```
Comment (JonasVanHoof) [inline data-access/mandataris.ts:860]: Just so the linter does not complain in the queryString

File: controllers/mandataris.ts [hunk 3]
```diff
@@ -203,3 +208,24 @@ export async function handleBulkSetPublicationStatus(
     STATUS_CODE.BAD_REQUEST,
   );
 }
+
+async function generate(config): Promise<Array<string>> {
```
Comment (Rahien) [inline controllers/mandataris.ts:212]: maybe call this `generateRows` to mirror the thing that's going on in the frontend. Else generate feels a bit unclear to me

**Review by Rahien — APPROVED** [review]

File: controllers/mandataris.ts (hunk 3 above)
Comment (JonasVanHoof) [reply controllers/mandataris.ts]: Updated it to _generateRows_

**Pull request merged**